src/App.tsx

```diff
@@ -714,25 +715,25 @@ function App(): ReactElement {
             onChangePalette={setCategoricalPalette}
           />
         </div>
-        <FlexRowAlignCenter className={styles.headerRight}>
-          <Button type="link" className={styles.copyUrlButton} onClick={openCopyNotification}>
-            <FlexRowAlignCenter $gap={6}>
+        <FlexRowAlignCenter $gap={12}>
+          <FlexRowAlignCenter $gap={2}>
+            <LoadDatasetButton onRequestLoad={handleLoadRequest} currentResourceUrl={collection?.url || datasetKey} />
+            <Export
+              totalFrames={dataset?.numberOfFrames || 0}
+              setFrame={setFrameAndRender}
+              getCanvas={() => canv.domElement}
+              // Stop playback when exporting
+              onClick={() => timeControls.pause()}
+              currentFrame={currentFrame}
+              defaultImagePrefix={datasetKey + "-" + featureName}
```

Comment:
personal preference :
```suggestion
              defaultImagePrefix={`${datasetKey}-${featureName}`}
```